docs/user/canonical-urls.rst

```diff
@@ -0,0 +1,53 @@
+Canonical URLs
+==============
+
+Canonical URLs allow you to have consistent page URLs across all your domains.
```

Comment:
I think this needs more explanation. What does it mean "consistent page URLs across all your domains" for a user?